app/src/renderer/vuex/modules/blocks.js

```diff
@@ -83,13 +90,25 @@ export default ({ node }) => {
       }
 
       commit(`setSyncing`, false)
+      // New RPC endpoint in sync, reset UI block list
+      commit(`setBlocks`, [])
```

Comment:
If we reconnect to the same node, we now clear all cached blocks. Does that make sense?

Reply:
good catch - probably not.